components/sysutils/nut/nut-bins.p5m

```diff
@@ -0,0 +1,111 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Jim Klimov
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)/bins@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY) - NUT Command-line binaries and scripts"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+<transform file path=usr.*/man/.+ -> default mangler.man.stability uncommitted>
+
+depend fmri=$(COMPONENT_FMRI)-common@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION) type=require
+
+### NUT Command-line binaries and scripts (daemons, clients)
+
+### Note: While nut-scanner can use snmp, neon/xml, ipmi and usb libs, it pulls
+### them truly dynamically (if available) and skips otherwise. Installing all
+### the NUT driver packages should pull those libs by harder ldd requirements.
+### Conversely if you don't want those UPSes (got no drivers) why scan them? :)
+
```

Comment:
No strong opinion... given that there is typically not much data to process (and LFS is enabled in recipe), 64-bit programs are not likely to be required to use (and as 64-bit, they have a downside in efficiency). My guess is that the current split is okay - people will get 32-bit, and if this ever causes problems, can refer to 64-bit ones as they are available.